docs/api/graphql/overview.md

```diff
@@ -0,0 +1,204 @@
+
+There's also an edges property that contains nodes. The data under node shows a proposal, information you would get by querying for a proposal by ID.
+
+Paginated results use this added nesting to provide the extra information you need to navigate the data. Each edge item in the array also has a cursor property you could select if you need to.
+
+After fetching this result, it's clear that there are more proposals. You can use `endCursor` from the first result to fetch the next one, which will return the next 1 proposal, and a new cursor:
+
+```
+{
+  proposalsConnection(pagination: { first: 1, after: "eyJzdGF0ZSI6NiwidmVnYV90aW1lIjoiMjAyMy0wNS0yNFQxNDowMDowMS4wMjk3MTZaIiwiaWQiOiI4NDAyNWU2ODM4N2NmNjFjMmI5MTIyOGQ3NjhkY2RkNGYxMGE5ZWU1Y2QyODI0ZmRlYTM1YjI1OTk3NmY1OWMxIn0="}) {
+    pageInfo {
+      hasNextPage
+      endCursor
+    }
+    edges {
+      node {
+        id
+        rationale {
+          title
+        }
+      }
+    }
+  }
+} 
+```
+
+## Subscriptions
+Subscriptions are like queries that keep returning data as it updates. These work similarly to the Observe endpoints in the REST service. 
+
+For example, the following query would show a result only when a new proposal came in:
+
+```
+subscription {
+  proposals {
+    id
+    rationale {
+      title
+    }
+  }
+}
+```
+
+A more useful query might be the following one, which receives an update every time the mid-price of the specified market updates:
+
+```
+subscription {
+  marketsData(marketIds: "2dca7baa5f7269b08d053668bca03f97f72e9a162327eebd941c54f1f9fb8f80") {
+    midPrice
+  }
+}
+```
+
+### Bus event subscription 
+The bus event subscription is different from other subscriptions in one particular way. It has an extra filter for batch size. If you choose 0, then the subscription will deliver each event as it occurs. Otherwise, if you pick 10, for example, you'll receive results in batches of 10, but no response until there are 10 results.
+
+Example that would provide an immediate result from the event bus query:
+
+```
+subscription {
+  busEvents(types: [TimeUpdate], batchSize: 0) {
+    event {
+      ... on TimeUpdate {
+        timestamp
+      }
+    }
+  }
+}
+```
+
+```
+
+{
+  "data": {
+    "busEvents": [
+      {
+        "event": {
+          "timestamp": "2023-03-05T15:05:33.177819749Z"
+        }
+      }
+    ]
+  }
+}
+```
+
+## GraphQL playground
+For a more approachable interface, GraphQL playground provides an interactive environment for you to build up queries with full integrated documentation. 
+
+For most users this will be the best way to get to grips with the schema. 
+
+Try visiting the GraphQL endpoint of a server. For example, you can try the [GraphQL Playground](https://api.n07.testnet.vega.xyz/graphql/) for Fairground, the Vega operated testnet.
+
+
```

Comment:
Data Node warns that GraphQL playground is not recommended for production environments:

```
2023-06-13T14:23:39.631+0100    WARN    datanode.gateway.gql    graphql/server.go:206   graphql playground enabled, this is not a recommended setting for production
```

It might be worth noting that this may not be available on mainnet, and recommend testing either on fairground or user running their own data-node. I'm not sure what the best policy would be.